server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/test/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutesTest.java

```diff
@@ -1618,15 +1612,18 @@ private InputStream fullContent(MessageResult messageResult) {
         return ImmutableList.copyOf(messageManager.getMessages(MessageRange.all(), FetchGroupImpl.MINIMAL, session));
     }
 
-    private DeletedMessage.Builder.RequireHasAttachment<DeletedMessage.Builder.FinalStage> messageWithAttachmentBuilder() {
+    private DeletedMessage buildMessageWithHasAttachment(boolean hasAttachment) {
```

Comment:
never, ever, add a boolean to a method to pass it to a builder, it defeats the builder pattern entirely.

Reply:
Okay, I just want to hide the size building stage